test/backward_compatibility/check_backward_compatibility.py

```diff
@@ -115,6 +115,7 @@
     ('aten::__or__', datetime.date(2020, 6, 30)),
     ('aten::__xor__', datetime.date(2020, 6, 30)),
     ('aten::split', datetime.date(2020, 6, 30)),
+    ('aten::mkldnn_convolution_backward_weights', datetime.date(2020, 6, 30)),
```

Comment:
Sorry you need to update this date before landing.

Reply:
changed.